server/user-groups.ts

```diff
@@ -344,3 +346,59 @@ export class GlobalAuth extends Auth {
 		return true;
 	}
 }
+
+/**
+ * userid:section
+ */
+class SectionLeaders extends Map<ID, RoomSection> {
```

Comment:
This seems to share a lot of code with globalAuth, especially regarding save/load. I'm fine with it but it might be something to look into refactoring in the future (having a `UserMap<T>` class or something).

Reply:
After some discussion, I think I strongly prefer this to just be a room rank in Staff rather than an entirely separate class with its own persistence mechanism.

Reply:
Any particular symbol?

Reply:
(We decided to go with `▸`, to represent a collapsed section.)